plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -601,29 +602,55 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase {
           .anchor(focalPoint)
           .build(),
         actionAfter = {
-          cameraAnimationsPlugin.anchor = scaleCachedAnchor
-          notifyOnScaleListeners(detector)
-          spanSinceLast = abs(detector.currentSpan - detector.previousSpan)
+          onScaleAnimationEnd(detector)
         }
       )
     } else {
       val zoomBy =
         ln(detector.scaleFactor.toDouble()) / ln(PI / 2) * ZOOM_RATE.toDouble() * internalSettings.zoomAnimationAmount.toDouble()
-      easeToImmediately(
-        CameraOptions.Builder()
-          .zoom(mapCameraManagerDelegate.cameraState.zoom + zoomBy)
-          .anchor(focalPoint)
-          .build(),
-        actionAfter = {
-          cameraAnimationsPlugin.anchor = scaleCachedAnchor
-          notifyOnScaleListeners(detector)
-          spanSinceLast = abs(detector.currentSpan - detector.previousSpan)
+      if (internalSettings.simultaneousRotateAndPinchToZoomEnabled) {
+        val zoom = cameraAnimationsPlugin.createZoomAnimator(
+          cameraAnimatorOptions(mapCameraManagerDelegate.cameraState.zoom + zoomBy) {
+            startValue(mapCameraManagerDelegate.cameraState.zoom)
+            owner(MapAnimationOwnerRegistry.GESTURES)
+          }
+        ) {
+          duration = 0
         }
-      )
+        zoom.addListener(
+          onEnd = {
+            onScaleAnimationEnd(detector)
+          }
+        )
+        val anchorAnimator = cameraAnimationsPlugin.createAnchorAnimator(
+          options = cameraAnimatorOptions(focalPoint) {
+            owner(MapAnimationOwnerRegistry.GESTURES)
+          },
+        ) {
+          duration = 0
+        }
+        cameraAnimationsPlugin.playAnimatorsSequentially(zoom, anchorAnimator)
```

Comment:
it's duration 0 so does not matter but logically they need to be played together and not sequentially.

Reply:
Yeah, let's change to `playAnimatorsTogether`